src/stylesheets/blocks/_footer.scss

```diff
@@ -0,0 +1,101 @@
+.footer {
+  background-color: color(deep-blue);
+  padding-top: rem(25);
+  padding-bottom: rem(80);
+  position: relative;
+  overflow: hidden;
+
+  @media (screen(lg)) {
+    padding-top: rem(50);
+  }
+
+  .container {
+    .sm-title,
+    .lg-text {
+      color: color(white);
+      @include font-smoothing;
+    }
+
+    .sm-title {
+      @media (screen(lg)) {
+        align-items: center;
+        display: flex;
+        flex-direction: row;
+        font-size: rem(16);
+        justify-content: center;
+        letter-spacing: 0.05em;
+      }
+    }
+    
+    .lg-text {
+      font-size: rem(14);
+      line-height: rem(24);
+
+      @media (screen(lg)) {
+        font-size: rem(15);
+      }
+    }
+
+    .footer__logo {
+      height: rem(50);
+      margin-top: rem(10);
+      width: rem(135);
+
+      @media (screen(lg)) {
+        height: rem(60);
+        margin-left: rem(20);
+        margin-top: 0;
+        width: rem(180);
+      }
+    }
+
+    &::after {
+      background: url('../../images/Florence_IconPrimary-01.svg') no-repeat center / contain;
+      bottom: rem(-150);
+      content: '';
+      height: rem(140);
+      position: absolute;
+      right: calc(50% - 80px);
+      width: rem(130);
+
+      @media (screen(lg)) {
+        height: rem(210);
+        width: rem(190);
+        right: unset;
+        left: rem(-15);
+        bottom: rem(-135);
+      }
+    }
+  }
+}
+
+.footer__btn,
+.footer__btn:visited {
+  align-items: center;
+  background-color: color(deep-blue);
+  border-radius: rem(100) rem(100);
+  border: rem(4) solid color(yellow);
+  color: color(yellow);
+  cursor: pointer;
+  display: flex;
+  font-size: rem(17);
+  height: rem(45);
+  justify-content: center;
+  letter-spacing: 0.05em;
+  margin: rem(30) auto rem(10) auto;
+  text-decoration: none;
+  text-transform: uppercase;
+  transition-duration: 0.1s;
+  transition-timing-function: ease-in-out;
```

Comment:
These two can be combined into the shorthand: `transition: 0.1s ease-in-out;`. Also you don't need to set this rule (and potentially all these other rules) for visited and hover together. You should be able to break them out like this:
```
.footer__btn {
  [common styles here, for example...]
  text-transform: uppercase;
  transition: 0.1s ease-in-out;
  width: rem(230);

  &:hover {
    [hover specific styles here (no transition necessary)]
  }

  &:visited {
    [visited specific styles here, if necessary (no transition necessary)]
  }
}
```